shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -421,16 +421,25 @@ - (void)loadView {
   _scrollView.reset(scrollView);
 }
 
+- (flutter::PointerData)generatePointerDataForFake {
+  flutter::PointerData pointer_data;
+  pointer_data.Clear();
+  pointer_data.kind = flutter::PointerData::DeviceKind::kTouch;
+  // `UITouch.timestamp` is defined as seconds since system startup. Synthesized events can get this
+  // time with `NSProcessInfo.systemUptime`. See
+  // https://developer.apple.com/documentation/uikit/uitouch/1618144-timestamp?language=objc
+  pointer_data.time_stamp = [[NSProcessInfo processInfo] systemUptime] * kMicrosecondsPerSecond;
+  return pointer_data;
+}
+
 static void SendFakeTouchEvent(FlutterEngine* engine,
                                CGPoint location,
                                flutter::PointerData::Change change) {
   const CGFloat scale = [UIScreen mainScreen].scale;
-  flutter::PointerData pointer_data;
-  pointer_data.Clear();
+  flutter::PointerData pointer_data = [[engine viewController] generatePointerDataForFake];
   pointer_data.physical_x = location.x * scale;
   pointer_data.physical_y = location.y * scale;
   pointer_data.kind = flutter::PointerData::DeviceKind::kTouch;
```

Comment:
nit: Can you remove this line of code? It's redundant now.

Reply:
My negligence...